toolkit/scripts/srpm_pack.mk

```
@@ -19,13 +19,19 @@ SRPM_BUILD_LOGS_DIR = $(LOGS_DIR)/pkggen/srpms
 toolchain_spec_list = $(toolchain_build_dir)/toolchain_specs.txt
 srpm_pack_list_file = $(BUILD_SRPMS_DIR)/pack_list.txt
 
-ifneq ($(strip $(SRPM_PACK_LIST)),)
+# Configure the list of packages we want to process into SRPMs
+custom_srpm_pack_list = $(strip $(SRPM_PACK_LIST))
```

Why not use "[override](https://www.gnu.org/software/make/manual/make.html#Override-Directive)" like we did with `REPO_LIST`? The word "custom" seems a bit misleading, like it's not related to the original `SRPM_PACK_LIST` or we modified to a greater degree than by simply stripping the whitespaces.
```suggestion
override SRPM_PACK_LIST = $(strip $(SRPM_PACK_LIST))
```

---

Switched to using override. Needed to be `override SRPM_PACK_LIST := ...` to avoid circular definition of the variable.